SDL/src/Main.cpp

```diff
@@ -1,22 +1,20 @@
 
 #include <cv.h>
 #include "opencv2/opencv.hpp"
-#include <SongLoader.h>
-#include <SongPlayer.h>
-
+#include "SongLoader.h"
```

Comment:
between SongLoader.h and SongPlayer.h? Sorry its hard to tell from this comment^